src/main/java/org/isf/stat/gui/ExamsList1Launcher.java

```diff
@@ -76,8 +73,7 @@ private void initialize() {
 		Dimension screensize = kit.getScreenSize();
 		pfrmBordX = (screensize.width / 3) - (pfrmExactWidth / 2);
 		pfrmBordY = (screensize.height / 3) - (pfrmExactHeight / 2);
-		this.setBounds(pfrmBordX,pfrmBordY,pfrmExactWidth,pfrmExactHeight);
-		
+		this.setBounds(pfrmBordX, pfrmBordY, pfrmExactWidth, pfrmExactHeight);
 		this.setContentPane(getJPanel());
```

Comment:
pack() + setLocationRelative(null) ?

Reply:
I changed it but it appears that this class is never used.   Maybe deleting it is the correct fix.

Reply:
like `DiseasesListLauncher`, it is directly launched from menu (`MNI_ID_A = examlist1`).

Thanks for changing it!